model/task_lifecycle.go

```diff
@@ -442,9 +442,21 @@ func MarkEnd(t *task.Task, caller string, finishTime time.Time, detail *apimodel
 	}
 
 	status := t.ResultStatus()
+	if t.DisplayOnly {
+		grip.Debug(message.Fields{
+			"task_id":  t.Id,
+			"status":   t.Status,
+			"EVG-7769": "marking display task as finished in MarkEnd",
+		})
+	}
 	event.LogTaskFinished(t.Id, t.Execution, t.HostId, status)
 
 	if t.IsPartOfDisplay() {
+		grip.Debug(message.Fields{
+			"exec_task": t.Id,
+			"task_id":   t.DisplayTask.Id,
```

Comment:
Add some nil checking for this since it doesn't come from the db

Reply:
IsPartOfDisplay only returns true if t.DisplayTask != nil